fileops-service/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "fileops-service.fullname" . }}
+  labels:
+    {{- include "fileops-service.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: {{ .Values.service.targetPort }}
```

Comment:
this is not defined in the values.yaml file. It definitely should.

Reply:
I have defined as

```
service:
  type: ClusterIP
  port: 5078
  targetPort: 5078
```